app/views/course/sections/school-placements/where-you-will-train-scitt.md

```diff
@@ -1,3 +1,3 @@
-You’ll be placed in different schools during your training. You can’t pick which schools you want to be in, but your course will try to place you in schools you can commute to.
+You’ll be placed in different schools during your training. You can’t pick which schools you want to be in, but your training provider will try to place you in schools you can commute to.
```

Comment:
Could insert the name of the training provider here? (“Training provider” feels slightly jargony and can be ambiguous if there’s both a university and a school partnership involved - although we’re also guilty of using it in places!)

Could also swap “commute” for “travel” perhaps? Maybe even either drop “try to” or add “travel to easily” - otherwise it implies there’s a possibility you might get placed somewhere it’s not even possible to commute to at all??

Reply:
I think I'll leave it as training partner for now, given the issues with the length of some provider names. We do use it in other places so hopefully won't cause confusion about what we mean by it. It's a term we'd certainly like to test more, though.

Yes, agree about your other point :) I won't include 'easily travel to', as that's a bit subjective. I'll keep it simple and just say 'will place you in schools you can travel to.'